go.mod

```diff
@@ -36,15 +38,17 @@ require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.1.0 // indirect
 	github.com/perimeterx/marshmallow v1.1.5 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
@leszko these updates were done to fix conflicts in the github.com/pkg/errors v0.9.1 dependency preventing the tests from running.